connexion/handlers.py

```diff
@@ -45,12 +45,11 @@ def handle(self, *args, **kwargs):
         """
         Actual handler for the execution after authentication.
         """
-        response = problem(
+        raise AuthenticationProblem(
```

Comment:
This is causing the error in `tests/aiohttp/test_aiohttp_api_secure.py::test_auth_all_paths[pyloop]` because if no middleware handles the raised exception, then aiohttp swallows the response converting it to a 500 error:
```
>       assert get_inexistent_endpoint.status == 404
E       assert 500 == 404
E         -500
E         +404
```
You might need something like #952 to add that middleware for aiohttp.

Reply:
Ahh okay, thanks! I will work on that today. I was confused why it was happening last night, but it was getting late for me..

It sort of makes sense now that you pointed it out.

Reply:
Resolving this since everything seems to be working now. Let me know if something is off.